atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -208,6 +208,16 @@ - (void)addItemToMenu:(NSMenu*)menu
 
   base::string16 role = model->GetRoleAt(index);
   atom::AtomMenuModel::ItemType type = model->GetTypeAt(index);
+
+  // allow for creating a services menu without supplying a submenu
+  if (role == base::ASCIIToUTF16("services")) {
+    [item setTarget:nil];
+    [item setAction:nil];
+    NSMenu* submenu = [[NSMenu alloc] initWithTitle:@"Services"];
```

Comment:
Is it ok to hardcode the "Services" menu title? Shouldn't it depend on a user's locale and be provided by the OS?